test/unit/streaming.controllers.BufferControllerSpec.js

```diff
@@ -446,12 +412,9 @@ describe('BufferController', function () {
             eventBus.trigger(Events.PLAYBACK_SEEKING);
         });
 
+        /*
```

Comment:
Also, not sure what to do here; doesn't seem like a valid test when the buffer is getting cleared out on seek?

Reply:
Let me take a look at this part. Structure of the test seems to be ok but I need to take a deeper look.